spring-boot-project/spring-boot-starters/README.adoc

```diff
@@ -182,7 +182,7 @@ do as they were designed before this was clarified.
 | https://github.com/okta/okta-spring-boot
 
 | https://www.optaplanner.org/[OptaPlanner]
-| https://github.com/kiegroup/optaplanner/tree/master/optaplanner-integration/optaplanner-spring-boot-starter
+| https://github.com/kiegroup/optaplanner/tree/master/optaplanner-spring-integration/optaplanner-spring-boot-starter (https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/spring-boot-school-timetabling[quickstart example])
```

Comment:
Please revert the quickstart link. None of the other entries have such a link and I am not keen to introduce that. You can make sure the quickstart guide is prominent in whatever link you provide here.